test/plots/piePlotTests.ts

```diff
@@ -264,6 +264,25 @@ describe("Plots", () => {
         });
         svg.remove();
       });
+
+      it("labels outside of the render area are hidden", () => {
+        piePlot.detach();
+        piePlot.removeDataset(simpleDataset);
+        var data = [5000, 5000, 5000];
```

Comment:
The label for the bottom slice is the first one to get forced out of the SVG (it's directly on an axis, as opposed to at a diagonal).